#Trayectorias en mundo

float32   x_UbicacionRob
float32   y_UbicacionRob
float32   theta_Rob

camina/Punto[6] trayectoriaPata

